src/components/Reactions/MiniQuickEmojiReactions.tsx

```diff
@@ -1,106 +1,94 @@
-import PropTypes from 'prop-types';
 import React, {useRef} from 'react';
 import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import type {Emoji} from '@assets/emojis/types';
 import BaseMiniContextMenuItem from '@components/BaseMiniContextMenuItem';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import * as EmojiUtils from '@libs/EmojiUtils';
 import getButtonState from '@libs/getButtonState';
 import * as EmojiPickerAction from '@userActions/EmojiPickerAction';
 import * as Session from '@userActions/Session';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
-import {baseQuickEmojiReactionsDefaultProps, baseQuickEmojiReactionsPropTypes} from './QuickEmojiReactions/BaseQuickEmojiReactions';
+import type {BaseQuickEmojiReactionsOnyxProps, BaseQuickEmojiReactionsProps} from './QuickEmojiReactions/types';
 
-const propTypes = {
-    ...baseQuickEmojiReactionsPropTypes,
+type MiniQuickEmojiReactionsOnyxProps = Omit<BaseQuickEmojiReactionsOnyxProps, 'preferredLocale'>;
```

Comment:
Why `preferredLocale` is omitted? I see that it is used in the component

Reply:
Yeah, but it's omitted from OnyxProps type, cause this component doesn't take `preferredLocale` from `withOnyx` HOC.
If it looks confusing I can update this type to be just
```
type MiniQuickEmojiReactionsOnyxProps = {
    /** All the emoji reactions for the report action. */
    emojiReactions: OnyxEntry<ReportActionReactions>;

    /** The user's preferred skin tone. */
    preferredSkinTone: OnyxEntry<string | number>;
};
```

Reply:
Yes, let's use a separate type 👍